src/components/va-alert/va-alert.tsx

```diff
@@ -48,6 +48,9 @@ export class VaAlert {
    */
   @Prop() closeable: boolean = false;
 
+  
+  @Prop() alertLoaded: Function = undefined;
```

Comment:
I think instead of making this a `@Prop()`, let's make this an [`@Event`](https://stenciljs.com/docs/events).

Then as far as naming conventions go, we're prefixing all our custom events with `va`. Because we might want to do this with other components, I'd like to make the name fairly generic so those other components can use it as well. Consistent naming reduces cognitive load on the developer using the component.

So what do you think of making this an `@Event: vaComponentDidLoad`?